app/models/package_sub_category.rb

```diff
@@ -0,0 +1,6 @@
+class PackageSubCategory < ActiveRecord::Base
+  belongs_to :package_type
+  belongs_to :package_category
+
+  validates :package_type_id, uniqueness: { scope: :package_category_id }
+end
```

Comment:
@swatijadhav my previous comment about whether we should rename to ```package_categories_package_types``` should have gone here. cc @mcm-ham